packages/cli/tests/e2e/e2e.test.ts

```diff
@@ -287,361 +282,25 @@ describe("e2e tests", function test() {
-        nonce: 1n,
-        pollId: 0n,
-        newVoteWeight: 9n,
-        maciAddress: maciAddresses.maciAddress,
-        salt: genRandomSalt(),
-        privateKey: users[2].privKey.serialize(),
-        signer,
-      });
-      await publish({
-        pubkey: users[3].pubKey.serialize(),
-        stateIndex: 4n,
-        voteOptionIndex: 2n,
-        nonce: 3n,
-        pollId: 0n,
-        newVoteWeight: 3n,
-        maciAddress: maciAddresses.maciAddress,
-        salt: genRandomSalt(),
-        privateKey: users[3].privKey.serialize(),
-        signer,
-      });
-      await publish({
-        pubkey: users[3].pubKey.serialize(),
-        stateIndex: 4n,
-        voteOptionIndex: 2n,
-        nonce: 2n,
-        pollId: 0n,
-        newVoteWeight: 2n,
-        maciAddress: maciAddresses.maciAddress,
-        salt: genRandomSalt(),
-        privateKey: users[3].privKey.serialize(),
-        signer,
-      });
-      await publish({
-        pubkey: users[3].pubKey.serialize(),
-        stateIndex: 4n,
-        voteOptionIndex: 1n,
-        nonce: 1n,
-        pollId: 0n,
-        newVoteWeight: 9n,
-        maciAddress: maciAddresses.maciAddress,
-        salt: genRandomSalt(),
-        privateKey: users[3].privKey.serialize(),
-        signer,
-      });
-    });
-
-    it("should generate zk-SNARK proofs and verify them", async () => {
-      await timeTravel({ ...timeTravelArgs, signer });
-      await mergeSignups({ ...mergeSignupsArgs, signer });
-      await genProofs({ ...genProofsArgs, signer });
-      await proveOnChain({ ...proveOnChainArgs, signer });
-      await verify({ ...(await verifyArgs()), signer });
-    });
-  });
-
-  describe("9 signups, 1 message", () => {
-    after(async () => {
-      await clean();
-    });
-
-    const users = [
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-      new Keypair(),
-    ];
-
-    before(async () => {
-      // deploy the smart contracts
-      maciAddresses = await deploy({ ...deployArgs, signer });
-
-      const startDate = await getBlockTimestamp(signer);
-
-      // deploy a poll contract
-      await deployPoll({ ...deployPollArgs, signer, pollStartDate: startDate, pollEndDate: startDate + pollDuration });
-    });
-
-    it("should signup nine users", async () => {
-      // eslint-disable-next-line @typescript-eslint/prefer-for-of
-      for (let i = 0; i < users.length; i += 1) {
-        // eslint-disable-next-line no-await-in-loop
-        await signup({ maciAddress: maciAddresses.maciAddress, maciPubKey: users[i].pubKey.serialize(), signer });
-      }
-    });
-
-    it("should join nine users", async () => {
+    it("should publish 4 messages", async () => {
       // eslint-disable-next-line @typescript-eslint/prefer-for-of
-      for (let i = 0; i < users.length; i += 1) {
+      for (let i = 0; i < 21; i += 1) {
```

Comment:
Why is it 21?

Reply:
it was to have just over one batch of votes